contrib/mesos/pkg/scheduler/service/service.go

```diff
@@ -344,7 +344,7 @@ func (s *SchedulerServer) AddStandaloneFlags(fs *pflag.FlagSet) {
 
 func (s *SchedulerServer) AddHyperkubeFlags(fs *pflag.FlagSet) {
 	s.addCoreFlags(fs)
-	fs.StringVar(&s.kmPath, "km-path", s.kmPath, "Location of the km executable, may be a URI or an absolute file path.")
+	fs.StringVar(&s.kmPath, "km-path", s.kmPath, "Location of the km executable, may be a URI or an absolute file path; if started with 'file://', the km in agent local path was used.")
```

Comment:
suggestion: "may be prefixed with 'file://' to specify the path to a pre-installed, agent-local km binary"